noir-projects/aztec-nr/aztec/src/macros/functions/call_interface_stubs.nr

```diff
@@ -19,24 +23,27 @@ comptime global FROM_FIELD: TypedExpr = {
 
 comptime global SERIALIZED_ARGS_NAME: Quoted = quote { serialized_args };
 
-pub comptime fn stub_fn(f: FunctionDefinition) -> Quoted {
-    let is_static_call = is_fn_view(f);
+pub comptime fn register_private_fn_stub(f: FunctionDefinition) {
```

Comment:
Newly introduced this function. Makes sense to go ahead and just register the stub here in the registry instead of in the `transform_private` function. This results in better encapsulation.